clients/dotnet-core-rest/src/Tika.RestClient/Features/ApiKeys/IApiKeysClient.cs

```diff
@@ -6,8 +6,8 @@ namespace Tika.RestClient.Features.ApiKeys
 {
     public interface IApiKeysClient
     {
-        Task<IEnumerable<ApiKey>> GetAllAsync();
-        Task<ApiKey> CreateAsync(ApiKeyCreate apiKeyCreate);
-        Task DeleteAsync(string key);
+        Task<IEnumerable<ApiKey>> GetAllAsync(string clusterId = null);
```

Comment:
default value for "default" keyword is "null", "" or 0, false, etc. Please assign to = default and not = null